composer.json

```diff
@@ -14,7 +14,13 @@
     },
     "autoload": {
         "psr-4": {
-          "Google\\GAX\\":"src/"
+          "Google\\GAX\\":"src/",
+          "google\\protobuf\\":"src/generated/",
+          "google\\api\\":"src/generated/",
+          "google\\rpc\\":"src/generated/",
+          "google\\type\\":"src/generated/",
+          "google\\longrunning\\":"src/generated/",
+          "google\\logging\\":"src/generated/"
```

Comment:
You are right, this won't work for all of the classes that don't match the file names. Have switched to use classmap instead. This requires running dump-autoload when classes are added to the src/generated directory, but this should be rare.